pep/app/pods/components/page/sidebar/widgets/favorites/component.ts

```diff
@@ -36,7 +55,7 @@ export default class PageSidebarWidgetsFavorites extends Component<PageSidebarWi
      */
     @action
     onElementChange() {
```

Comment:
missed this before, but we should rename this to be more accurate, maybe `onPreferencesUpdate()`

Reply:
Thats partly correct. This function is both called on `did-insert` and `did-update`. When running on `did-insert` the preferences were not updated

Reply:
Yeah I missed that. in that case, maybe `onInsertOrPrefsUpdate()` or even just two separate actions, `onElementInsert()` and `onPreferencesUpdate()` that both call the load method. This might be ideal because if/when we add diff checking for the IDs array, that should only happen in the `did-insert` path.